src/glue/jobs/s3_to_json.py

```diff
@@ -544,8 +563,21 @@ def main() -> None:
                 Key = message["source_key"]
         )
         s3_obj["Body"] = s3_obj["Body"].read()
+        cohort = None
+        if "adults_v1" in message["source_key"]:
```

Comment:
Will we ever have v2 or more cohorts? Do we feel like this is something we should parse out?

Reply:
hmm is that something we would want some say over? The name of the S3 key prefix containing the cohort's data

Reply:
Not necessarily. It can be an arbitrary key prefix, and as long as it stays consistent we can filter for it and partition accordingly.